src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ResponseAcceptationContentProvider.java

```diff
@@ -37,14 +40,23 @@ public ResponseAcceptationContentProvider(
                 "courtDecision"
             ));
             PaymentIntention courtPaymentIntention = courtDetermination.getCourtPaymentIntention();
-            content.putAll(paymentIntentionContentProvider.createContent(
-                courtPaymentIntention.getPaymentOption(),
-                courtPaymentIntention.getRepaymentPlan().orElse(null),
-                courtPaymentIntention.getPaymentDate().orElse(null),
-                "The agreed amount",
-                "courtPaymentIntention"
-                )
-            );
+            Optional<LocalDate> paymentDate = courtPaymentIntention.getPaymentDate();
+            if (courtPaymentIntention.getPaymentOption() == PaymentOption.BY_SPECIFIED_DATE
+                && paymentDate.isPresent()
+                && paymentDate.get().equals(LocalDate.of(9999, 12, 31))) {
```

Comment:
pls add a comment as to why this date has special meaning

Reply:
done